qiskit/circuit/parameter.py

```diff
@@ -165,10 +164,10 @@ def __hash__(self):
     # operation attempts to put this parameter into a hashmap.
 
     def __getstate__(self):
-        return (self._name, self._uuid, self._symbol_expr)
+        return (self._uuid, self._symbol_expr)
```

Comment:
This change might affect code using picking. Would it be possible to keep the tuple shape the same?

Reply:
should be good for another look